adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -95,16 +101,14 @@
     private static final int ACCOUNT_MANAGER_ERROR_CODE_BAD_AUTHENTICATION = 9;
     private static final int MAX_RESILIENCY_ERROR_CODE = 599;
 
-    @Override
-    protected void setUp() throws Exception {
-        super.setUp();
+    @Before
+    public void setUp() throws Exception {
 
         Log.d(TAG, "setup key at settings");
-        getContext().getCacheDir();
-        System.setProperty("dexmaker.dexcache", getContext().getCacheDir().getPath());
+        System.setProperty("dexmaker.dexcache", InstrumentationRegistry.getContext().getCacheDir().getPath());
         Log.d(TAG, "setup key at settings");
-        getContext().getCacheDir();
-        System.setProperty("dexmaker.dexcache", getContext().getCacheDir().getPath());
+        InstrumentationRegistry.getContext().getCacheDir();
+        System.setProperty("dexmaker.dexcache", InstrumentationRegistry.getContext().getCacheDir().getPath());
```

Comment:
this is original code https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/dev/adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java 

What I did here is replacing the API called. But it seems it's redundant since the 1st day this file was created. I would remove the redundant. Nice catch